src/Config/Converters/EntityRestOptionsConverter.cs

```diff
@@ -100,7 +100,12 @@ public override void Write(Utf8JsonWriter writer, EntityRestOptions value, JsonS
     {
         writer.WriteStartObject();
         writer.WriteBoolean("enabled", value.Enabled);
-        writer.WriteString("path", value.Path);
+
+        if (value.Path is not null)
+        {
+            writer.WriteString("path", value.Path);
+        }
```

Comment:
Is it possible to have this property written by default? and if null, abide by the JsonSerializer option `DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull` to not write out the property?

Reply:
That's a good question, this might not be needed. I'll verify and remove if it isn't.

Reply:
It'll be needed as we are owning how the properties end up in the `writer` in this case, so we can still write nulls, even if the `DefaultIgnoreCondition` says not to.

I've changed this so that we respect what the `DefaultIgnoreCondition` is regarding `null`, so that if we change it in the future the code should adapt.